app/config/base_config.yml

```diff
@@ -104,6 +104,9 @@ liip_imagine:
         media_library_one_image_large:
             filters:
                 thumbnail: { size : [1600, 500], mode : inbound }
+        content_block_image:
```

Comment:
it would be nice if it was possible to configure this in the module, something to look at but doesn't need to change now

Reply:
Not sure what you mean, but have you added this somewhere? like an issue or and todo?